docs/safety.md

```diff
@@ -0,0 +1,186 @@
+- Make sure all are clear before using tool
+- Ensure item you are drilling into is clamped in some way so that it
+  doesn’t go flying or break
+- Use step bits when possible for drilling into plastic
+
+### When wiring to a battery (if battery leads are exposed):
+
+- Make sure all are clear before removing tape / no one is touching
+  other side of battery to prevent shorts
+- If anything is loose or hanging use electric tape to prevent any
+  accidental shorts
+- Use battery boots (when possible)
+- If battery shorts, STOP what you're doing, turn off all systems to
+  prevent further damage, identify problem.
+
+### When soldering:
+
+- Ensure some sort of fan/filter is on (either the ceiling air sucker or
+  the mini fan)
+- Ensure soldering iron is not left on when finished
+
+### When using Heat-Gun
+
+- Make sure ONLY pointing at items want to heat up
+- Keep others clear when heat gun is on
+- Ensure heat gun is turned off when finished
+
+### When running new steel cables for Brake-by-Cable:
+
+- Use swage tool in vice when possible
+- Clean all steel shavings when cutting cables
+- Tape/cover all sharp ends of the cable
+
+## Test Day Procedures
+
+### Test day safety officer
+
+- One person (can be volunteer) is assigned to be safety officer for day
+- This person will make sure people are following the ALL CLEAR shouts
+- Person makes sure no one is in car when people are under car
+- Person makes sure people using tools are wearing goggles
+- Will be the person to call 911 if needed
+- Will be the first person to use the fire extinguisher if needed
+- Responsible for asking people not being safe to leave the room/test
+  environment
+
+### In General
+
+- If not directly touching / testing something on car, STAY CLEAR
+- If have curiosity questions, it’s okay to ask but wait until the team
+  member who is working is finished or save questions for reflections or
+  other times
+- If afraid you might forget your question, write it down somewhere
+- Ideally, unless it’s urgent, don’t distract people using tools and
+  working on car
+- Not only is it important for team members to focus while using tools
+  to work on the car, distractions can cause mistakes which could cause
+  the car to become unsafe
+
+### When car is ARMED
+
+- Armed means any of the following: Key is in ignition, Node boards are
+  powered
+- Take extreme care not to short connections
+- If possible, turn off car / nodes before using metal tools on the
+  boards (metal snips, screw drivers, etc)
+- If anything shorts / makes smoke/ explodes, STOP what you're doing,
+  call for help, turn all systems OFF to prevent further damages,
+  identify problem before moving on
+- If working with something connected to a battery, take even more care
+  not to short components
+
+### When outside:
+
+- Yellow caution tape and orange cones for to corral/block pedestrians
+- No one in front or behind car while car is “ARMED” NO EXCEPTIONS
+- Must have fire extinguisher in car
```

Comment:
```suggestion
*When outside of 41CS:*

- Secure the test area with caution tape and orange cones
- Have a megaphone in case an announcement needs to be made
```